apps/web/components/DestinationCalendarSelector.tsx

```diff
@@ -50,6 +50,21 @@ const DestinationCalendarSelector = ({
         value: `${cal.integration}:${cal.externalId}`,
       })),
     })) ?? [];
+  const customStyles = {
```

Comment:
Taking the opportunity to fix colors in the destination calendar selector as well